packages/indexer-common/src/indexer-management/actions.ts

```diff
@@ -12,13 +15,61 @@ import { Eventual, join, Logger, timer } from '@graphprotocol/common-ts'
 export class ActionManager {
   constructor(
     public allocationManager: AllocationManager,
+    public networkMonitor: NetworkMonitor,
+    private currentEpoch: number,
+    private maxAllocationEpoch: number,
```

Comment:
some alternative to passing in these parameters I was thinking were we could pass in `network: Network` and call contract functions, or implement similar `network.contracts` functions?

Reply:
I think it's a good idea to implement functions to get epoch config data in `NetworkMonitor`.